vmdb/app/controllers/ops_controller/diagnostics.rb

```diff
@@ -1164,4 +1160,9 @@ def diagnostics_build_tree
     TreeBuilderOpsDiagnostics.new("diagnostics_tree", "diagnostics", @sb)
   end
 
+  def build_supported_depots_for_select
```

Comment:
Sorry, many comments...
- Is @protocols_arr used anywhere else?
- Each with push could just be collect
- Overall it looks like this could just be ```@supported_depots_for_select = FileDepot.supported_depots.values.sort```